optimum/exporters/openvino/__main__.py

```diff
@@ -54,25 +58,29 @@ def infer_task(
     revision: Optional[str] = None,
     cache_dir: str = HUGGINGFACE_HUB_CACHE,
     token: Optional[Union[bool, str]] = None,
+    library_name: Optional[str] = None
 ):
     task = TasksManager.map_from_synonym(task)
     if task == "auto":
-        try:
-            task = TasksManager._infer_task_from_model_name_or_path(
-                model_name_or_path=model_name_or_path,
-                subfolder=subfolder,
-                revision=revision,
-                cache_dir=cache_dir,
-                token=token,
-            )
-        except KeyError as e:
-            raise KeyError(
-                f"The task could not be automatically inferred. Please provide the argument --task with the relevant task from {', '.join(TasksManager.get_all_tasks())}. Detailed error: {e}"
-            )
-        except RequestsConnectionError as e:
-            raise RequestsConnectionError(
-                f"The task could not be automatically inferred as this is available only for models hosted on the Hugging Face Hub. Please provide the argument --task with the relevant task from {', '.join(TasksManager.get_all_tasks())}. Detailed error: {e}"
-            )
+        if library_name == 'open_clip':
+            task = "zero-shot-image-classification"
+        else:
+            try:
+                task = TasksManager._infer_task_from_model_name_or_path(
+                    model_name_or_path=model_name_or_path,
+                    subfolder=subfolder,
+                    revision=revision,
+                    cache_dir=cache_dir,
+                    token=token,
+                )
+            except KeyError as e:
+                raise KeyError(
+                    f"The task could not be automatically inferred. Please provide the argument --task with the relevant task from {', '.join(TasksManager.get_all_tasks())}. Detailed error: {e}"
```

Comment:
TaskManager return list of tasks without open_clip or not? Should it be explicitly included

Reply:
TaskManager.get_all_tasks() include zero-shot-image-classification task